.storybook/preview.js

```diff
@@ -34,6 +37,7 @@ export const parameters = {
   chakra: {
     theme,
   },
+  layout: "centered",
```

Comment:
Global positioning of all the stories to be in the center of the preview container

Can also be added on a separate PR if desired.